### PR TITLE
Update matterbridge.toml

### PR DESCRIPTION
**Pull request by Naleksuh**

### CODE REVIEW

**Comment by MacFan4000** [comment]
To clarify this is to ignore WikiAuthBot (discord bot).
This has my support as an IRC moderator, and NotAracham's support as a discord moderator, though may be good to get support from another moderator.

File: modules/profile/templates/matterbridge.toml [hunk 1]
```diff
@@ -141,6 +141,18 @@
     account ="irc.libera"
     channel = "#miraheze-meetings"
     
+[[gateway]]
+    name = "mh.wikicreators"
+    enable = true
+    
+    [[gateway.inout]]
+    account = "discord.miraheze"
+    channe l= "ID:1029629996664365126"
```
Comment (MacFan4000) [inline modules/profile/templates/matterbridge.toml:150]: There is an extra space here where there shouldn’t be

**Comment by MacFan4000** [comment]
This also now adds a relay for the wiki creator channels

**Review by RhinosF1 — APPROVED** [review]

**Comment by Universal-Omega** [comment]
This also has my support as both a Discord and IRC Moderator.

**Pull request merged**